Problem1.java

```diff
@@ -0,0 +1,20 @@
+class Solution {
+    public int[] searchRange(int[] nums, int target) {
+        //1. Brute force - sort through each element
```

Comment:
Please complete the solutions as soon as you can. Contact the TAs or the professor if you have any difficulties.